internal/mocks/inmem/ccipreader_inmem.go

```diff
@@ -138,7 +138,7 @@ func (r InMemoryCCIPReader) GetChainFeePriceUpdate(
 }
 
 func (r InMemoryCCIPReader) DiscoverContracts(
-	ctx context.Context, destChain cciptypes.ChainSelector,
+	ctx context.Context, allChains []cciptypes.ChainSelector,
```

Comment:
`allChains` might be a bit misleading. What is `all`?

Reply:
The OffRamp has a function to get source chain configurations by chain selector, it has all configurations regardless of whether or not the current node supports them. So currently the full list needs to come from outside of the ccip reader.

The onchain team is hopefully going to address this by providing a `getSourceChainSelectors()` function which would replace this parameter.